Common/Utilities.cs

```diff
@@ -2804,14 +2793,10 @@ public static void CreateCertificate(string domainName, string pfxName, string c
             }
             else
             {
-                //File.Copy(
-                //    Path.Combine(Utilities.ProjectPath, "Asset", "SampleTestCertificate.pfx"),
-                //    Path.Combine(Utilities.ProjectPath, "Asset", pfxName),
-                //    overwrite: true);
             }
         }
 
-        public static void UploadFileToWebApp(IPublishingProfile profile, string filePath, string fileName = null)
+        public static async Task UploadFileToWebAppAsync(IPublishingProfile profile, string filePath, string fileName = null)

```

Comment:
Heaths has reviewed this in another PR-Azure-Samples/key-vault-dotnet-manage-key-vaults#8
we decide to use await for async code, followed by .ConfigureAwait(false).